src/ingestor.ts

```diff
@@ -101,6 +101,14 @@ export default async function ingestor(req) {
     const id = snapshot.utils.getHash(body.data);
     let payload = {};
 
+    if (body.hash && body.hash !== id) {
```

Comment:
In which case do we pass `hash` in the body?

Reply:
I have no idea, seems like it’s optional

Reply:
hmm don't see it any where else, maybe we can remove this if condition